src/index.html

```diff
@@ -9,10 +9,51 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
     />
```

Comment:
The attributes of the link tag should start on a new line with 2-space indentation relative to the tag. The closing bracket should be on the same level as the opening one.